easy_my_coop/models/company.py

```diff
@@ -92,6 +92,9 @@ def _compute_base_logo(self):
         translate=True,
         help="Text to display aside the checkbox to approve financial risk.",
     )
+    send_certificate_email = fields.Boolean(string="Send certificate email", default=True)
+    send_confirmation_email = fields.Boolean(string="Send confirmation email", default=True)
+    send_capital_release_email = fields.Boolean(string="Send Capital Release email", default=True)
```

Comment:
Can you add in roadmap.rst "gather and consolidate all easy_my_coop settings in  the application parameters"?

Reply:
Sorry @robinkeunen but I can't find the file `roadmap.rst` in this module. Can you please point me to the file?

I only find a `ROADMAP.rst` file in the `easy_my_coop_api` module.

Reply:
You can create the file in the `readme` directory :-)

Reply:
Done! :man_dancing: